docs/Troubleshooting/Troubleshooting.md

```diff
@@ -5,10 +5,49 @@ description: Frequently asked questions FAQ and answers for troubleshooting Pant
 
 If Pantheon is not working as expected, here are some things to check or try. 
 
-## Command Line Options
+## Supplied Genesis Block Does Not Match Stored Chain Data 
 
-Characters such as smart quotes and long (em) hyphens won't work in Pantheon command line options. Ensure quotes have
-not been automatically converted to smart quotes or double hyphens combined into em hyphens.
+If you get a `Supplied genesis block does not match stored chain data` error, delete the `database` directory in 
+the node [data directory](../Reference/Pantheon-CLI-Syntax.md#data-path). 
+
+## Host Not Authorized 
+
+If you are receiving a `Host not authorized` error when attempting to access the JSON-RPC API, ensure [`--host-whitelist`](../Reference/Pantheon-CLI-Syntax.md#host-whitelist)
+includes the host from which you are sending the RPC or `*`. 
+
+## Ports
```

Comment:
Peers fail to connect (rather than Ports?)
Maybe get them to check enode addresses in staticnodes/bootnodes against the text output during remote node startup?